src/Domain/Bootstrap.php

```diff
@@ -130,17 +130,22 @@ function() {
 	 * Define the global feature flag.
 	 */
 	protected function define_feature_flag() {
-		$allowed_flags = [ 'experimental', 'stable' ];
-		$flag          = getenv( 'WOOCOMMERCE_BLOCKS_PHASE' );
-		if ( ! in_array( $flag, $allowed_flags, true ) ) {
-			if ( file_exists( __DIR__ . '/../../blocks.ini' ) ) {
-				$woo_options = parse_ini_file( __DIR__ . '/../../blocks.ini' );
-				$flag        = is_array( $woo_options ) && 'experimental' === $woo_options['woocommerce_blocks_phase'] ? 'experimental' : 'stable';
-			} else {
-				$flag = 'stable';
-			}
+		if ( defined( 'WC_BLOCKS_IS_FEATURE_PLUGIN' ) ) {
+			$default_flag = '2';
+		} else {
+			$default_flag = '1';
 		}
-		define( 'WOOCOMMERCE_BLOCKS_PHASE', $flag );
+
+		$allowed_flags = [ '1', '2', '3' ];
```

Comment:
Is there a reason we use strings here instead of numbers?

Reply:
Environment variables and ini values are treated as strings, to avoid casting and recasting things all over the place, I assume everything is a string and when I define the flag I turn it to an int.